### ISSUE TITLE
Admonitions are not being rendered in Docusaurus based sites

### ISSUE THREAD

**Issue by amimas** [favoloso/remarkable-admonitions#3]
Hello,

I came across this repository through the discussion in following issue in Docusaurus repository:

https://github.com/facebook/Docusaurus/issues/1238

The preview in the readme of this repository looks very nice and I want try it out but unfortunately it's not working for me. Not sure if I'm misconfigured something or if this is a bug. In my case, the admonitions/callouts are not being rendered by docusaurus. They are showing up as plain text. For example, here's the snippet of my markdown:

```markdown
> **Note**: You can resize a pane by clicking and dragging right, left, up, or down on the division frames between each pane. You can also quickly resize a pane by clicking the minimize and maximize buttons in the upper right of each pane.

:::tip
You can resize a pane by clicking and dragging right, left, up, or down on the division frames between each pane. You can also quickly resize a pane by clicking the minimize and maximize buttons in the upper right of each pane.
:::
```

Here's how it's being rendered. Note that I'm trying to convert the standard  markdown "blockquote" type message using admonitions from this plugin. The blockquote is still being rendered but not the admonition.

![image](https://user-images.githubusercontent.com/16211767/56146843-061eb980-5f75-11e9-8e8e-310375c12b90.png)

I'm using the latest version of Docusaurus (`1.8.0`). I've added this package following the instructions and here's the snippet from my `package.json` file:

```json
  "devDependencies": {
    "docusaurus": "^1.8.0"
  },
  "dependencies": {
    "remarkable-admonitions": "^0.2.1"
  }
```

Next, I updated `siteConfig.js` of my docusaurus site by adding following into the `siteConfig` object.

```js
  // Add custom markdown plugins
  markdownPlugins: [
    // Highlight admonitions (callouts such as tips, warnings, note, important, etc)
    require('remarkable-admonitions')({ icon: 'svg-inling' })
  ],
```

Then I added the css file from `https://github.com/favoloso/remarkable-admonitions/blob/master/styles/docusaurus-admonitions.css` to my `website/static/custom/` directory.

Finally I updated my markdown file as shown above. I tried to move the downloaded css file to different locations (i.e. `website/static/css`, `website/static`) but none of that seemed to help.

**Comment by amimas** [comment]
I just found where I made a mistake. It was in the `siteConfig.js` where I had a typo. I entered the markdown plugin's configuration as `icon: 'svg-inling'`. I should have been `icon: 'svg-inline'`.

Found the error when I restarted my docusaurus site because it complained this error:

```
Error: Icon Renderer "svg-inling" is not defined
```

Not sure why I didn't get this error before as I restarted the site several times.

Closing this issue as it's a false alarm. Thanks for building this plugin btw.
